extensions/ws-security/runtime/src/main/java/io/quarkiverse/cxf/ws/security/CustomAlgSuiteLoader.java

```diff
@@ -0,0 +1,111 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package io.quarkiverse.cxf.ws.security;
+
+import org.apache.cxf.Bus;
+import org.apache.cxf.ws.security.policy.custom.AlgorithmSuiteLoader;
+import org.apache.neethi.Policy;
+import org.apache.wss4j.policy.SPConstants;
+import org.apache.wss4j.policy.model.AbstractSecurityAssertion;
+import org.apache.wss4j.policy.model.AlgorithmSuite;
+import org.eclipse.microprofile.config.Config;
+
+/**
+ * This class retrieves a custom AlgorithmSuite for use with restricted security policies
+ */
+public class CustomAlgSuiteLoader implements AlgorithmSuiteLoader {
+
+    public static final String DIGEST_ALGORITHM = "http://www.w3.org/2001/04/xmlenc#sha256";
```

Comment:
@ppalaga I'd like to access default values of CxfBuilTimeConfig class here to avoid duplicated constant creation. Do you see any other way of avoiding duplicates? (I don't see any buildTimeConfig for ws-security, where it would be probably avoidlable. (core-deployment does not see ws-secuurity/runtime and vice versa)